examples/auth_example/auth_example_server/README.md

```diff
@@ -4,12 +4,65 @@ This is the starting point for your Serverpod server.
 
 To run your server, you first need to start Postgres and Redis. It's easiest to do with Docker.
 
-    docker compose up --build --detach
+```bash
+cd auth_example/auth_example_server
```

Comment:
Suggestion: Since we use this for our documentation, might as well fix it here as well?

```suggestion
$ cd auth_example/auth_example_server
```